plugins/jobs/src/js/components/Breadcrumbs.js

```diff
@@ -0,0 +1,120 @@
+import { Link } from "react-router";
+import prettycron from "prettycron";
+import PropTypes from "prop-types";
+import React from "react";
+import { Tooltip } from "reactjs-components";
+
+import TaskStates from "#PLUGINS/services/src/js/constants/TaskStates";
+import Breadcrumb from "#SRC/js/components/Breadcrumb";
+import BreadcrumbSupplementalContent from "#SRC/js/components/BreadcrumbSupplementalContent";
+import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
+import Icon from "#SRC/js/components/Icon";
+import PageHeaderBreadcrumbs from "#SRC/js/components/PageHeaderBreadcrumbs";
+
+function ItemStatus({ item: { jobRuns } }) {
+  if (!jobRuns) {
+    return null;
+  }
+  const { longestRunningActiveRun } = jobRuns;
+  if (
+    !longestRunningActiveRun ||
+    !longestRunningActiveRun.tasks.longestRunningTask
+  ) {
+    return null;
+  }
+  const { status } = longestRunningActiveRun.tasks.longestRunningTask;
+
+  return (
+    <BreadcrumbSupplementalContent>
+      <div className="service-page-header-status muted">
+        ({TaskStates[status].displayName})
+      </div>
+    </BreadcrumbSupplementalContent>
+  );
+}
+
+function ItemSchedule({ item: { schedules } }) {
+  if (!schedules || !schedules.nodes.length || !schedules.nodes[0].enabled) {
```

Comment:
Could be rewritten using `findNestedPropertyInObject` as well, though in this place it's not critical at all.